.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,7 +1,34 @@
-_Issue #, if available:_
+<!--
+Please make sure to read the Pull Request Guidelines:
+https://github.com/aws-amplify/amplify-codegen/blob/master/CONTRIBUTING.md#pull-requests
+-->
 
-_Description of changes:_
 
-_How are these changes tested:_
+#### Description of changes
+<!--
+Thank you for your Pull Request! Please provide a description above and review
+the requirements below.
+-->
 
-By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
+
+#### Issue #, if available
+<!-- Also, please reference any associated PRs for documentation updates. -->
+
+
+
+#### Description of how you validated changes
+
+
+
+#### Checklist
+<!-- Remove items that do not apply. For completed items, change [ ] to [x]. -->
+
+- [ ] PR description included
+- [ ] `yarn test` passes
+- [ ] Tests are [changed or added](https://github.com/aws-amplify/amplify-codegen/blob/master/CONTRIBUTING.md#tests)
+- [ ] Relevant documentation is changed or added (and PR referenced)
+- [ ] Breaking changes to existing customers are released behind a feature flag or major version update
+- [ ] Changes are tested using Sample Applications for all relevant platforms (iOS/android/flutter/Javascript)
```

Comment:
Do we have some official sample apps for testing? How can third-party developers find these apps?

Reply:
We do not have one right now. Once we have them, we could direct them to the sample apps for reference. For now, they will have to create a sample app to minimally test the feature being added/modified. Updated this accordingly.